imports/zones/client.lua

```diff
@@ -4,6 +4,7 @@ local glm = require 'glm'
 ---@field id number
 ---@field coords vector3
 ---@field distance number
+---@field type string
```

Comment:
String literal type `---@field type 'poly' | 'sphere' | 'box'`.
Might need to mark debug as private and add the setDebug method too.

Reply:
After marking debug as private
![image](https://github.com/overextended/ox_lib/assets/82737367/9753e051-57e6-4a45-a56a-743287c14a66)